### PR TITLE
Add GeoJSON as GetFeatureInfoFormat (deegree 3.6)

### LINKED ISSUE #1582: Add support for Json output format to WMS GetFeatureInfo
deegree WMS GetFeatureInfo operation shall support for Json/GeoJson as output format.

### PR DESCRIPTION
**Pull request by lgoltz**
Example configuration:

```
<wms:GetFeatureInfoFormat>
  <wms:GeoJSON allowOtherCrsThanWGS84="true" allowExportOfGeometries="true" />
  <wms:Format>application/geojson</wms:Format>
</wms:GetFeatureInfoFormat>
```

Query Parameters:
 * `GEOMETRIES=true` to export geometries (if `allowExportOfGeometries="true"`)
 * `INFO_CRS=EPSG:25832` to set the CRS of the geometries in the response (if `allowOtherCrsThanWGS84="true"`)

Fixes #1582

### CODE REVIEW

File: deegree-core/deegree-core-featureinfo/src/main/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializer.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package org.deegree.featureinfo.serializing;
```
Comment (stephanr) [inline deegree-core/deegree-core-featureinfo/src/main/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializer.java:1]: This file is missing the necessary Java file header (see https://github.com/deegree/deegree3/wiki/Developer-guidelines#java-file-headers)

File: deegree-core/deegree-core-featureinfo/src/test/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializerTest.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
+package org.deegree.featureinfo.serializing;
```
Comment (stephanr) [inline deegree-core/deegree-core-featureinfo/src/test/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializerTest.java:1]: This file is missing the necessary Java file header (see https://github.com/deegree/deegree3/wiki/Developer-guidelines#java-file-headers)

**Review by stephanr — CHANGES_REQUESTED** [review]
See comments above

File: deegree-core/deegree-core-featureinfo/src/main/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializer.java (hunk 1 above)
Comment (lgoltz) [reply deegree-core/deegree-core-featureinfo/src/main/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializer.java:26]: Added header with https://github.com/deegree/deegree3/pull/1623/commits/39e39c1de94d59f8a026218a93ba07e3b66dd4d9

File: deegree-core/deegree-core-featureinfo/src/test/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializerTest.java (hunk 2 above)
Comment (lgoltz) [reply deegree-core/deegree-core-featureinfo/src/test/java/org/deegree/featureinfo/serializing/GeoJsonFeatureInfoSerializerTest.java:26]: Added header with https://github.com/deegree/deegree3/pull/1623/commits/39e39c1de94d59f8a026218a93ba07e3b66dd4d9

**Review by stephanr — APPROVED** [review]

**Review by tfr42 — APPROVED** [review]

**Pull request merged**